src/gui/properties/piecesbar.cpp

```diff
@@ -255,11 +252,7 @@ void PiecesBar::showToolTip(const QHelpEvent *e)
     QString toolTipText;
     QTextStream stream(&toolTipText, QIODevice::WriteOnly);
     bool showDetailedInformation = QApplication::keyboardModifiers().testFlag(Qt::ShiftModifier);
-    if (!showDetailedInformation) {
-        stream << simpleToolTipText();
-        stream << '\n' << tr("Hold Shift key for detailed information");
```

Comment:
I mean insert `return` here and then drop `else` and put its content one indentation level up.